01-docker-terraform/README.md

```diff
@@ -1,8 +1,8 @@
 ### Introduction
 
-* [Video](https://www.youtube.com/watch?v=-zpVha7bw5A)
+* [Video: "Data Engineering Zoomcamp 2023"](https://www.youtube.com/watch?v=-zpVha7bw5A)
```

Comment:
Perhaps it's also worth adding 2024 launch video here? What do you think?

Reply:
Good call - done! (I added couple other links as well - see most recent commit)